hbw/config/datasets.py

```diff
@@ -73,6 +86,7 @@ def hbw_dataset_names(config: od.Config, as_list: bool = False) -> DotDict[str:
     dataset_names = DotDict.wrap({
         # **data_datasets,
         "data": data_datasets,
+        "data_jethtmet": config.x.if_era(cfg_tag="is_for_sf", values=data_jethtmet_datasets),
```

Comment:
I think this always needs to be some Iterable, but at the moment would be `None` when `is_for_sf` is not set.
```suggestion
        "data_jethtmet": [*config.x.if_era(cfg_tag="is_for_sf", values=data_jethtmet_datasets)],
```

Reply:
never mind.